col/build.sbt

```diff
@@ -19,21 +19,17 @@ generateHelpersTask := {
     src / "C.scala",
   )
 
-  /*
-  val compile = FileFunction.cached(streams.value.cacheDirectory / "antlr4", FilesInfo.hash, FilesInfo.hash)(changedSet => {
+
+  val compile = FileFunction.cached(streams.value.cacheDirectory / "removeThisToGenerate-src_managed", FilesInfo.hash)(changedSet => {
```

Comment:
This code will check if the folder "removeThisToGenerate-src_managed" is changed. If that is the case it will regenerate the Scala classes.

Reply:
Ah, right, I see why it was changed from antlr now. Is the approach now automatic or do you have to manually remove the folder to trigger regeneration of the class files?

Reply:
To be honest. I am not that familiar with build.sbt files. So I think you have to manually remove it. I am not sure what the problem was with an automatic approach.

Reply:
This is in fact automatic, it is just incidentally also cached. If for whatever reason the caching mechanism fails, you can indeed *also* remove this directory to trigger re-generation.